src/core/hooks/useRemoteList.ts

```diff
@@ -52,6 +52,10 @@ export default function useRemoteList<
     }
   }
 
+  if (!remoteList) {
+    return [];
+  }
```

Comment:
This is not the desired behaviour.

If there is no remote list, it's because it hasn't loaded yet. If it hasn't loaded yet, it probably (at this point) has started loading, and there is a promise. In that case, the promise should be thrown, to suspend rendering until the data has loaded.

Currently this actually causes a bug. Going into a location where households have not yet loaded will trigger the households to load, and in the meantime will return an empty array. That empty array is interpreted by the `households2` page as meaning that there are no households, and opens it in the "floorplan editor state".